src/test/java/rest/gorestcoIn/utils/User.java

```diff
@@ -0,0 +1,36 @@
+package rest.gorestcoIn.utils;
+
+import io.restassured.path.json.JsonPath;
+
+public class User {
+    private String name;
+    private String email;
+    private String gender;
+    private String status;
+
+    public User(String body){
```

Comment:
What is the meaning of giving JSON formatted String as a parameter, then extract it and assign to the instance variables? instead, you can organize your random user creation directly in `User.class`